tests/ssh-banner-only/test.rules

```diff
@@ -1,4 +1,4 @@
 alert ssh any any -> any any (ssh.software; content:"OpenSSH"; sid:1;)
 # broken?
```

Comment:
what was supposed to be broken here @catenacyber ?

Reply:
oops, I forgot to remove that line :x

Reply:
I did not do that test...
I guess we can be happy with `ssh.software`
There is one old keyword, and a new one for the sticky buffer thing if I remember correctly...

Reply:
Now I'm wondering if this rule was left there to check if the deprecated keyword still worked...